examples/example_charge.py

```diff
@@ -6,10 +6,11 @@
 metadata	          hash	Optional	Developer defined key value pairs
 redirect_url	    string	Optional	Redirect URL
 """
+import os
 
 from coinbase_commerce.client import Client
 
-API_KEY = "API_KEY"
+API_KEY = os.environ.get("COINBASE_COMMERCE_API_KEY", "API_KEY")
```

Comment:
While I think this is helfpul, I think we should not do this incase somebody is using production keys in their environment variables - they accidentally try an example and make a destructive change that they did not want to do. We should let users playing around with the examples choose how to pick the API KEY for examples, and not default to envars.

Reply:
Ok, I revert this commit

Reply:
Thank you!